application/Dockerfile

```
@@ -9,6 +9,7 @@ RUN python -m pip install --upgrade --no-cache-dir \
     pip>=20.1.1 \
     setuptools>=49.1.0 \
     wheel>=0.34.2
+RUN python -m pip wheel -w /wheels -r requirements.txt
```

Per @jezdez suggestion in https://github.com/mozilla/burnham/pull/78#issuecomment-704132772
```suggestion
RUN python -m pip wheel -w /wheels -r requirements.txt .
```